### ISSUE TITLE
Support for MBot in gazebo

### ISSUE THREAD

**Issue by fqez** [JdeRobot/JdeRobot#807]

**Comment by fqez** [comment]
First step:

I've managed to load a 3D model of the MBot robot into gazebo. 

![imagen](https://cloud.githubusercontent.com/assets/8070644/26237849/13550b4a-3c77-11e7-89c2-ccc8944946e4.png)

The next step is split the model into pieces (wheels, board, sensors, etc) and join them in a sdf format file. Then, it's time to add the plugins such as motors, pose3D, ultrasonic sensor, IR sensor, etc.

**Comment by fqez** [comment]
The new model is now up. You can see this video showing how it works in Gazebo.

https://youtu.be/xQ3IqUnNCP8

In this video, the MBot is using the default plugin for motors, which uses no physics, but the next iteration must cover a new plugin for the MBot at least.

The next step is to add more sensors to the robot, as now it only has motors and pose3d. Ultrasonic sensor and FollowLine sensor are desirable.

**Comment by fqez** [comment]
The new version of MBot is ready. It's still uncomplete, but I've added three more sensors. Besides the motors and pos3d, I've included:

 * Sonar sensor: There is an ultrasonic sensor working with the manufacturer specifications:
      Min range: 3cm (0.03m)
      Max range: 400cm (4.00m)
      Cone angle: 30º (0.523 rad)

 * Camera sensor: I've included a camera simulating the raspberry pi camera.
 * IR followline sensor: Also, there are a IR sensor for 'following the line' exercises. It is simulated by two cameras which resolution is 2x2px. With each image, I change the colorspace of it from BGR to HSV to get the brightness values (V from HSV), and then I calculate the histogram of V to be able to get the mean value. This way, when the camera sees black (meaning that it's seeing the line) the brightness will be very low,  and viceversa. This is not working yet due to problems with shadows that I'm still managing, next iteration will solve this.

 * Motors are still the same, but the next iteration will improve them.
 * Pose3D doesn't change.

The PR #1032 also adds compatibility for sonar ICE interface in JdeRobotComm.

The next iteration is centered in improve the motors plugin, and add a little grip with one servo.

Here is a video showing how it works: https://youtu.be/_k17V5pxrd8
